src/coloris.js

```diff
@@ -62,6 +62,9 @@
           }
           break;
         case 'theme':
+          if (options.autoDark !== false && !options.theme.includes('dark') && window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+            options.theme += '-dark';
+          }
           picker.className = `clr-picker clr-${options.theme.split('-').join(' clr-')}`;
```

Comment:
Can you rename autoDark to preferDark? I think it's clearer that way. What do you think?

Reply:
The user preferes dark (which is set in the OS), but this option is to automatically select the dark theme. I think we should have something with auto in the name.

Reply:
But I must say that I am not a native speaker. @melloware may I ask for your opinion?